compiler/compile/OSRData.hpp

```diff
@@ -119,7 +119,7 @@ class TR_OSRCompilationData
                            int32_t symRefOrder, int32_t symSize, bool takesTwoSlots);
    void ensureSlotSharingInfoAt(const TR_ByteCodeInfo& T);
    void addInstruction(TR::Instruction* instr);
-   void addInstruction(int32_t instructionPC, TR_ByteCodeInfo &bcInfo);
+   void addInstruction(int32_t instructionPC, TR_ByteCodeInfo bcInfo);
```

Comment:
Please explain why a reference is wrong - your justification here is the type is wrong - it isn't wrong you are choosing to change from a ref to an actual bcInfo. Is it that we can possibly modify it?